hie-base/Haskell/Ide/Engine/PluginTypes.hs

```diff
@@ -18,8 +18,14 @@ module Haskell.Ide.Engine.PluginTypes
   , ParamValP(..)
   , ParamMap
   , pattern ParamTextP
+  , pattern ParamIntP
+  , pattern ParamBoolP
   , pattern ParamFileP
```

Comment:
There is only one position type, and I plan to convert all the remaining FilePaths to Iris. We need these to signify that files and positions are allowed parameters to the plugins